tests/test_manifests.py

```diff
@@ -25,14 +25,22 @@ def test_product_manifest_properties():
     product = ProductFactory(product_variants=["1"])
     version = ProductVersionFactory(product_variants=["1"])
     stream = ProductStreamFactory(product_variants=["1"])
+    stream.save()
     variant = ProductVariantFactory(name="1")
     pnode = ProductNode.objects.create(parent=None, obj=product, object_id=product.pk)
     pvnode = ProductNode.objects.create(parent=pnode, obj=version, object_id=version.pk)
     psnode = ProductNode.objects.create(parent=pvnode, obj=stream, object_id=stream.pk)
     _ = ProductNode.objects.create(parent=psnode, obj=variant, object_id=variant.pk)
 
     build = SoftwareBuildFactory(build_id=1)
-    component = ComponentFactory(software_build=build)
+    build.save()
+    component = ComponentFactory(
+        software_build=build,
+        type="SRPM",
```

Comment:
Does your latest filter code exclude SRPMs when listing components? You might try removing the type="SRPM" line you added here and seeing if that makes the test pass.

In general, I'm happy to help review or debug things, but this doesn't really look like a manifesting bug to me. It's kind of hard to say for sure though, since I never reviewed (never got tagged) on the MR that implemented the latest filter.

In future you might tag me as a reviewer on other MRs too. There's no need to wait for approval from me before merging, but tagging me sends me an email notification, so will keep me aware of MRs submitted and new code added, which helps me give you a better answer when debugging instead of just guessing :)

Reply:
I got a notification on this MR. I meant that I never got tagged / never got to review your previous MR that added the "latest" filter, so I'm not sure what it's doing and why the data it returns isn't correct.